sse2neon.h

```diff
@@ -566,7 +566,12 @@ FORCE_INLINE uint8x16x4_t _sse2neon_vld1q_u8_x4(const uint8_t *p)
     +------+------+------+------+------+------+-------------+
  */
 
-/* Constants for use with _mm_prefetch.  */
+/* 
+Constants for use with _mm_prefetch.
+See https://github.com/gcc-mirror/gcc/blob/master/gcc/config/i386/xmmintrin.h#L39
```

Comment:
Don't reference code on GitHub since the line number might be changed. Instead, always cite Intel materials.